### ISSUE TITLE
can we  test only one image by pre-trained model, not the whole test set？

### ISSUE THREAD

**Issue by forbiddenname** [KaihuaTang/Scene-Graph-Benchmark.pytorch#32]
## ❓ Questions and Help
Thanks for your wonderful work!
i have a question.  can we  test only one image by pre-trained model, not the whole test set？
if so, what part of code should i change or what file should i prepare?

**Comment by ZhuYun97** [comment]
Do you have any ideas?

**Comment by forbiddenname** [comment]
> Do you have any ideas?
you can use /demo/predictor.py  to do it.

**Comment by ZhuYun97** [comment]
> > Do you have any ideas?
> > you can use /demo/predictor.py  to do it.

But it only used to visualize the results of the detector not for the scene parser.

**Comment by forbiddenname** [comment]
in the line 329, predictions = self.compute_prediction(image), it returns a BoxList, which includes <pred_labels, pred_scores, boxbounding, rel_pair_idxs, pred_rel_labels and pred_rel_scores>, not just the results of the detector.

**Comment by forbiddenname** [comment]
> > > Do you have any ideas?
> > > you can use /demo/predictor.py  to do it.
> 
> But it only used to visualize the results of the detector not for the scene parser.

in the line 329: predictions = self.compute_prediction(image), it returns the results of the object  detector and relation detector, not just object  detector.

**Comment by Ankit-Vohra** [comment]
Did you try this?
